pdk/1.0/modules/ROOT/pages/policies-pdk-create-schema-definition.adoc

```diff
@@ -271,6 +294,51 @@ properties:
       default: {"id": "1", "name": "John Doe"}
 ----
 
+[[dataweave-expressions]]
+=== DataWeave Expressions
+
+The `dataweave` format for `string` type parameters specifies the input parameter as a DataWeave expression.
+
+The `dataweave` variable supports the following expressions bindings:
+
+[source,yaml]
+----
+<configuration-parameter>:
+ type: string
+ format: dataweave
+ bindings:
+   payload:
+     mimeType: text
+   attributes: true
+   authentication: false
+   vars: []
+----
+
+* `payload`: The body of the imcoming request. Define the `mineType` as either `text` or `json` to set how to read the payload. the default `mimeType` is `text`. However, you must include the payload variable, for example, the following schema definition fails when applied to the API instance:
```

Comment:
1. mineType -> mimeType
2. the following schema definition fails when applied to the API instance if the dw expression references the payload. Eg: `#[payload]`
(That example is what you must write if you wish to restrict the usage of the payload in the dw expression )